OpenTrons/opentrons-api
# fix(app): fix the display condition for extension mount

# Overview
When the robotType is OT-2 Standard, the app doesn't show extension mount


https://github.com/Opentrons/opentrons/assets/474225/72d07ddf-b2e1-4d65-a6dd-5aa57b09aae2

The featureFlag will be removed in a following PR.

close RQA-1259



# Test Plan
1. open the Desktop app
2. turn on enable extended hardware
3. check a protocol for ot-2 and flex


# Changelog
- update rendering conditional 
- update test


# Review requests



# Risk assessment
low
